### PR TITLE
NAS-129104 / 24.10 / Fixed warning text

### PR DESCRIPTION
**Pull request by RehanY147**

### CODE REVIEW

File: src/app/pages/apps/components/dockerhub-rate-limit-info-dialog/dockerhub-rate-limit-info-dialog.component.ts [hunk 1]
```diff
@@ -20,9 +19,6 @@ export class DockerHubRateInfoDialogComponent {
   get warningText(): string {
     return this.translate.instant(
       this.helpText.dockerHubRateLimit.message,
```
Comment (denysbutenko) [inline src/app/pages/apps/components/dockerhub-rate-limit-info-dialog/dockerhub-rate-limit-info-dialog.component.ts:21]: now we can use this inside the template directly

**Review by denysbutenko — CHANGES_REQUESTED** [review]

File: src/app/pages/apps/components/dockerhub-rate-limit-info-dialog/dockerhub-rate-limit-info-dialog.component.ts [hunk 2]
```diff
@@ -16,18 +12,4 @@ import { DockerHubRateLimit } from 'app/interfaces/dockerhub-rate-limit.interfac
 })
 export class DockerHubRateInfoDialogComponent {
   helpText = helptextApps;
```
Comment (denysbutenko) [inline src/app/pages/apps/components/dockerhub-rate-limit-info-dialog/dockerhub-rate-limit-info-dialog.component.ts:14]: unused

**Review by denysbutenko — CHANGES_REQUESTED** [review]
ChartWizard needs to be updated since `MAT_DIALOG_DATA` was removed  
https://github.com/truenas/webui/blob/f3df7c398e53e8f35c88021bb65a14df9ec14ffa/src/app/pages/apps/components/chart-wizard/chart-wizard.component.ts#L523-L532

**Review by denysbutenko — APPROVED** [review]
Thanks, looks good

**Comment by undsoft** [comment]
backport

**Pull request merged**